docs/global/modules/core/pages/howto/usage-model-client-v2.adoc

```diff
@@ -0,0 +1,105 @@
+= How-To use the `model-client` v2 (Kotlin)
+:navtitle: Use the `model-client` v2 (Kotlin)
+
+NOTE: In order to use the client, you will need a xref:core:howto/usage-model-server.adoc[running model-server] to connect to.
+
+== Gradle Dependency
```

Comment:
We should somehow decide on how to do casing. I'm always more in favor of NOT using title case, which is less recognizable to humans than normal sentences

Reply:
yes, once we decide we have to rework all available docs